react/src/components/ServiceLauncherPageContent.tsx

```diff
@@ -837,14 +874,19 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
                       {
                         required: true,
                         min: 0,
-                        max: 10,
+                        max:
+                          user_resource_policy?.max_session_count_per_model_session ||
+                          10,
```

Comment:
Please consider that if `user_resource_policy?.max_session_count_per_model_session` is `0`, the maximum will be set to 10 in the current code. Is this right?

Reply:
If the value is 0, please use 0 as the max value in the UI. ([ref](https://teams.microsoft.com/l/message/19:3c0acc0825024daeb2211f55c4e7f4aa@thread.skype/1727686340501?tenantId=13c6a44d-9b52-4b9e-aa34-0513ee7131f2&groupId=74ae2c4d-ec4d-4fdf-b2c2-f5041d1e8631&parentMessageId=1727418901786&teamName=devops&channelName=Backend.AI%20Talks&createdTime=1727686340501))

Reply:
Yep. For error handling, I'll set it to follow the default value of 10 if the value is undefined or null.